contracts/CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ## Unreleased
 
-...
+### Changed
+
+- (Functions): Make billing regsitry's timeoutRequest pausable (#8299)
```

Comment:
typo

Reply:
Not a typo, releng requested it be added for 0.6.1's contract package release.
Just putting it in here to avoid a separate PR.

Reply:
Releng requested it for the 0.6.1 contracts release.
I can add to a separate PR if you think it matters.